pkg/util/equality/podset_test.go

```diff
@@ -28,9 +28,10 @@ import (
 
 func TestComparePodSetSlices(t *testing.T) {
 	cases := map[string]struct {
-		a         []kueue.PodSet
-		b         []kueue.PodSet
-		wantEqual bool
+		a                 []kueue.PodSet
+		b                 []kueue.PodSet
+		ignoreTolerations bool
+		wantEqual         bool
```

Comment:
```suggestion
		want         bool
```
nit: or `wantResult`, or `wantEquivalent`. We check equivalence which is a slightly more relaxed concept than "equal"